code/modules/admin/verbs/randomverbs.dm

```diff
@@ -910,3 +910,82 @@ Traitors and the like can also be revived with the previous role mostly intact.
 		to_chat(usr, "\red ERT has been <b>Disabled</b>.")
 		log_admin("Admin [key_name(src)] has disabled ERT calling.")
 		message_admins("Admin [key_name_admin(usr)] has disabled ERT calling.", 1)
+
+/client/proc/cmd_admin_mentorpai()
+	set category = "Admin"
+	set name = "Send Teacher pAI"
+	set desc = "Turns a ghost into a pAI that teaches a player"
+	if(!holder || !holder.rights || !holder.rights & R_ADMIN)

```

Comment:
no no no fuck no
never ever check rights directly, use `check_rights(R_ADMIN)` instead of all this